bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -137,13 +137,23 @@ <h3 class="c-sticky-cta-title-sub">{{ _('Live your life. Own your life.') }}</h3
       {% if switch('firefox_concert_series') %}
         {{ card(
           tag_label=_('Firefox Concert Series | Phosphorescent'),
-          title=_('Here’s to you. Here’s to the internet.'),
-          ga_title='Here’s to you. Here’s to the internet.',
+          title=_('Here’s to you. Here’s to live music.'),
+          ga_title='Here’s to you. Here’s to live music.',
           desc=_('Here’s to watching it free, from all kinds of artists, right from Firefox. Rock out with Phosphorescent, LIVE on Dec. 11.'),
           image_url='home/2018/cards/nov/concerts.jpg',
           aspect_ratio='mzp-has-aspect-1-1',
           link_url=url('firefox.concerts')
         )}}
+      {% elif switch('firefox_concert_series') and switch('firefox-concert-one-over') %}
```

Comment:
The first if statement in this series is matching so it's not going on to try to match the elif.

Reply:
ah of course, I need to reorder them... will fix.